var/spack/repos/builtin/packages/graphviz/package.py

```diff
@@ -116,6 +116,8 @@ class Graphviz(AutotoolsPackage):
     depends_on('sed', type='build')
     depends_on('libtool', type='build')
     depends_on('pkgconfig', type='build')
+    # to process f-strings used in gen_version.py
+    depends_on('python@3.6:', type='build')
```

Comment:
I assume this is a pretty recent addition; so to support older installations that might be using older pythons (for whatever reason) can you restrict this with `when='@a.bc:'`?

Reply:
Sorry, yes you are right. I have added the restriction.